packages/xdl/src/Android.ts

```diff
@@ -20,6 +20,8 @@ import * as Versions from './Versions';
 import { getUrlAsync as getWebpackUrlAsync } from './Webpack';
 
 let _lastUrl: string | null = null;
+let _isAdbOwner: boolean | null = null;
```

Comment:
Could we store this in the local `.expo` project settings? Or is that overkill @brentvatne

Reply:
i think the risk there is that the situation on the users' machine changes and then we have some cached value that's no longer accurate